content/blog/getting-started-with-deno/puppeteer-with-deno.md

```diff
@@ -0,0 +1,217 @@
+
+The next time we run this script (with the `--diff` flag) it will compare with the previously stored screenshots and will generate an image like the following:
+
+![diff-image](./diff-picsum-Mobile.png)
+
+And that's it!
+
+We have a simple program that will let us check if a website has changed since the last time we checked.
+
+We can now use another Deno command (available in the toolchain) to have this utility at hand whenever we need it - the script installer.
+
+## Using Deno's script installer
+
+To conclude, we'll explore one tool Deno includes in its binary, the script installer, made available by the `install` command.
+
+This is one of many tools like a test runner, linter, code formatter that you get access to after installing Deno.
+
+The `install` command will wrap any Deno program in an executable bash program and add it to `/usr/bin`, making it available across the system.
+
+```bash
+$ deno install --name website_compare --allow-read --allow-write --allow-net --allow-env --allow-run --unstable mod.ts
+Check file:///Users/alexandre/dev/personal/deno/puppeteer/mod.ts
+✅ Successfully installed website_compare
+/Users/alexandre/.deno/bin/website_compare
+```
+
+*Note how we're running the `install` command with the `--name` flag to set the name of the executable. We're also setting the permissions the script needs to execute, this is the only time we'll need to do it.*
+
+After doing this, we can run the `website_compare` wherever we are in our system.
+
+```bash
+$ website_compare https://picsum.photos
+Diff stored at /Users/alexandre/dev/personal/deno/puppeteer/screenshots/diff-picsum-mobile.png
+```
+
+And this completes our objective for the blogpost!
+
+We've demonstrated how can we use Deno's simplicity to make it even easier to write *puppeteer* scripts.
+
+As you've seen everything works the same as in Node.
+
+The big difference is that we can take advantage of some Deno advantages. Those are things like native TypeScript support, a clean standard-library, no *node_modules* and fine-grained permission control.
+
+## Conclusion
+
+Today we've explored `deno-puppeteer`, a package that makes it possible to write *puppeteer* scripts on Deno.
+
+Scripting is one of many use cases for Deno, and one of the reasons Deno was created, but definitely not the only one. In this blogpost series' we'll explore different parts of Deno, from rust interoperability to static site generation.
+
+We truly believe that, by building on the shoulders of giants (TypeScript, Node.js and Rust) and adding a few things to the mixture, Deno has all it takes to stand out.
+
+If you're interested in knowing more about Deno and how to use it to build tools and web applications, make sure you checkout my recently launched book [Getting started with Deno](). In the book, we'll carefully explain all the mentioned Deno features (and many others) while building real-world applications.
```

Comment:
Just as a reminder that the link to the book is missing, maybe even have a microwebsite created by you presenting the book, redirecting people to the appropriate amazon store (.com, .es, .de, .it, .fr, .com.br, .co.uk, etc.) enabling you to collect more analytics and metrics.

Reply:
Yeap, that's an ongoing work in another PR #24 but one that isn't finished yet! Thanks for the suggestion!